common-rendering/src/components/keyEvents.tsx

```diff
@@ -201,10 +203,19 @@ const timeStyles = (supportsDarkMode: boolean): SerializedStyles => css`
 `;
 
 const ListItem = ({ keyEvent, theme, supportsDarkMode }: ListItemProps) => {
+	useTimeAgo();
+	const time = timeAgo(new Date(keyEvent.time).getTime());
+
 	return (
 		<li css={listItemStyles(supportsDarkMode)}>
 			<div css={timeTextWrapperStyles}>
-				<time css={timeStyles(supportsDarkMode)}>{keyEvent.time}</time>
+				<time
+					dateTime={keyEvent.time}
```

Comment:
If keyEvent.time is an invalid date string for example `2021-15-09` the result would be `NaN NaN` 
Is that something we need to consider? Or we can just be sure that the consumer would always give up a correct date string?

Reply:
I think we'd have to assume that the date would always be in the correct format, I'm happy to listen to suggestions though if anyone thinks otherwise.